src/util/tenor-utils.ts

```diff
@@ -130,85 +130,87 @@ type tenorMediaObject = {
 
 //
 
-const DEFAULT_RESPONSE_LIMIT = 100;
-const TWELVE_HOURS = 43_200_000;
+const DEFAULT_RESPONSE_LIMIT: number = 50;
```

Comment:
The `DEFAULT_RESPONSE_LIMIT` was also changed to reflect the limit put by Tenor.